DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -158,6 +160,32 @@ public async Task AddResponseForUserAsync(int eventKey, ulong userId, string res
             }
         }
 
+        public async Task AddResponseBatchAsync(IEnumerable<ResponseReaction> reactions)
```

Comment:
Are these reactions meant to be all for the same message? Or could they be for different messages? 🙂

Reply:
Yeah these reactions could be for any event message, from any user, and of any type. This service will sort them into the relevant groups and put it together in one big batch 🙂